### ISSUE TITLE
Investigate technologies API performance

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/tech-report-apis#21]
On https://httparchive.org/reports/techreport/drilldown, I'm consistently seeing the [technologies](https://dev-gw-2vzgiib6.ue.gateway.dev/v1/technologies) endpoint taking about 3 seconds to respond.

<img width="1253" alt="image" src="https://github.com/HTTPArchive/tech-report-apis/assets/1120896/39583418-a683-4e38-abb2-2f5186e5d035">

This blocks the main "Technology" dropdown UI from showing the list of technologies, and it gets stuck only having one "ALL" option.

<img width="1224" alt="image" src="https://github.com/HTTPArchive/tech-report-apis/assets/1120896/0f75dfb6-0956-40cd-9722-dbb4358e908c">

Is there anything we can do to speed up the response?

**Comment by tunetheweb** [comment]
Can we also add cache-control headers to these? Even for a short while so it doesn't need to hit the server every single time when you're browsing around the site.

**Comment by rviscomi** [comment]
@maceto have you had a chance to look at this?

**Comment by tunetheweb** [comment]
Yeah definitely seems faster now. Sure you didn't fix it already as was definitely slow before :-)

**Comment by max-ostapenko** [comment]
As we discussed in a call, the next performance step can be connecting the backend to GLB and CDN.
